ui/src/pages/playground/playground.tsx

```diff
@@ -0,0 +1,11 @@
+import DrawingArea from '@components/DrawingArea';
+
+export default function Playground() {
+  return (
+    <>
+      <div className="mx-auto h-[750px] w-3/4">
```

Comment:
W ogóle nie reaguje na procentowe wartości wysokości, trzeba na sztywno.

Reply:
to dlatego że masz `min-height` a nie `height` przez to nie ma na podstawie czego obliczać procentów